client/components/operations/label.js

```diff
@@ -87,12 +87,18 @@ class LabelComponent extends React.Component {
     render() {
         if (!this.state.editMode) {
             return (
-                <button
-                  className="form-control text-left btn-transparent"
-                  id={ this.props.operation.id }
-                  onClick={ this.handleClickEditMode }>
-                    { this.buttonLabel() }
-                </button>
+                <div >
+                    <span className="text-uppercase shown-xs label-button">
```

Comment:
Can you use `visible-xs` instead of `shown-xs`? (and remove the code related to `shown-xs` in the style.css file)